src/assets/content/disclaimer.md

```diff
@@ -0,0 +1,29 @@
+---
+slug: '/disclaimer'
+date: '2021-01-23'
+title: 'Disclaimer'
+---
+
+**Be safe & secure:** [We highly recommend that you read our guide on How to Prevent Loss & Theft for some recommendations on how to be proactive about your security.](https://support.mycrypto.com/staying-safe/protecting-yourself-and-your-funds)
+
+**Always backup your keys:** MyCrypto.com & MyCrypto CX are not "web wallets". You do not create an account or give us your funds to hold onto. No data leaves your computer / your browser. We make it easy for you to create, save, and access your information and interact with the blockchain.
```

Comment:
**Always backup your keys:** MyCrypto.com & the MyCrypto Desktop Application are not "web wallets". You do not create an account or give us your funds to hold onto. No data leaves your computer / your browser. We make it easy for you to create, save, and access your information and interact with the blockchain.